schema.graphql

```diff
@@ -77,4 +81,33 @@ type Settlement @entity {
   firstTradeTimestamp: BigInt!
   "Collection of trades"
   trades: [Trade!] @derivedFrom(field: "settlement")
-}
\ No newline at end of file
+}
+
+# stores for USD calculations
+type Bundle @entity {
+  "Singleton id 1 containing ethPriceUSD value"
+  id: ID!
+  "Price of ETH in usd"
+  ethPriceUSD: BigDecimal!
+}
+
+type UniswapPool @entity {
```

Comment:
if there will be V2 and V3, maybe you will need to add the version? not sure

Reply:
The concern is for when you merge the 2 protocols in the same project

Reply:
We don't need v2 pools. We can fetch them by a contract call. So, that entity will be conditional and deployed only if we are V3 contracts.

Reply:
We need to do some work here to handling the merging the of the 2 protocols and make it configurable depending on what protocol/network we are deploying.